src/transformers/utils/quantization_config.py

```diff
@@ -587,3 +604,42 @@ def post_init(self):
             major, minor = compute_capability
             if major < 8:
                 raise ValueError("LLM-AWQ backend is only supported on GPUs with compute capability >= 8.0")
+
+        if self.do_fuse and self.fuse_max_seq_len is None:
+            raise ValueError(
+                "You cannot enable fused modules without specifying a `fuse_max_seq_len`, make sure to pass a valid `fuse_max_seq_len` for your usecase"
+            )
+
+        if self.do_fuse:
+            awq_version_supports_fusing = False
+            MIN_AWQ_VERSION = "0.1.7"
+            if is_auto_awq_available():
+                # For some reason `version.parse(importlib.metadata.version("awq"))` always returns
+                # `<Version('0.1.0')>` which makes that logic unusable. Therefore we need to import
+                # awq and get `awq.__version__`
+                import awq
+
+                awq_version = awq.__version__
+                awq_version_supports_fusing = version.parse(awq_version) >= version.parse(MIN_AWQ_VERSION)
```

Comment:
@casper-hansen do you have an idea why `importlib.metadata.version("awq")` always gives me `0.1.0` ?

Reply:
I have no idea, `0.1.0` is nowhere to be found in AutoAWQ.

https://github.com/casper-hansen/AutoAWQ/blob/main/awq/__init__.py#L1

Is it picking up llm-awq?

https://github.com/mit-han-lab/llm-awq/blob/main/pyproject.toml

Reply:
Indeed this works ! Thanks very much @casper-hansen !